exec/java-exec/src/test/resources/jsoninput/is_empty_tests.json

```
@@ -0,0 +1,27 @@
+[
+  {
+    "numeric_col": 1.3,
+    "text_col": "text",
+    "list_col": [
+      "v1", "v2"
+    ],
+   "map_column": {
+    "field1": "value1",
+    "field2": "value2",
+    "nested_map": {
+      "nested_field1": "nested_value1",
+      "nested_field2" : "nested_value2"
+      }
+    }
+  },{
+    "numeric_col": 2.3,
+    "text_col": "",
+    "list_col": [],
+    "map_column": {}
+  },{
+    "numeric_col": null,
+    "text_col": null,
+    "list_col": [],
+    "map_column": {}
+  }
```

```suggestion
  }, {
    "numeric_col": 1
  }
```
Can we add an object that is outright missing some queried properties and check that the is_empty unit tests pick those up as being empty?

---

@jnturton Done!  I addressed your review comments.  Thanks for the quick review!